app/controllers/api/v1/packages_controller.rb

```diff
@@ -273,6 +277,10 @@ def stock_serializer
         Api::V1::StockitItemSerializer
       end
 
+      def browse_serializer
+        Api::V1::BrowsePackageSerializer
```

Comment:
can we try not using this serializer ? I'm still not convinced it should exist in the first place :)

Reply:
I'll check and let you know about that.

Reply:
I've checked it, it works fine with `Api::V1::PackageSerializer`, so will use that only.

Reply:
I've checked it, it works fine with `Api::V1::PackageSerializer`, so will use that only.

cc - @patrixr @steveyken @namrataukirde

Reply:
nice : )

Reply:
great if we can remove `BrowsePackageSerializer`